game/lib/Main_menu.py

```diff
@@ -0,0 +1,79 @@
+import pygame
+import pygame.freetype
+import json
+import os
+from lib import colors
+from lib import gamestates
+from lib import button
+
+game_config = None
+file_exists = os.path.isfile("lib/config/game_config.json")
+if file_exists:
+    with open("lib/config/game_config.json") as json_file:
+        game_config = json.load(json_file)
```

Comment:
Zmień ścieżkę na "config/game_config.json" bo tam mają być wszystkie config i plik game_config odrazu przenieś bo to akurat moje niedopatrzenie że tak jest xD